### ISSUE TITLE
What are the details of the data in small_data.pkl?

### ISSUE THREAD

**Issue by IkumaUchida** [zhengwang125/play2vec#2]
Nice to meet you, I am currently working on a Reproduction of this paper using the script you wrote.
In the script, you have hard coated `small_data.pkl` as input data, what are the details of this data?

Stats perform's data was 7500 sequences of tracking data, will `small_data.pkl `be a reduced number of data than that?

I would appreciate it if you could tell me as much as you remember.

**Comment by zhengwang125** [comment]
Thanks for your interest. I am not sure, it may a reduced number of data than 7500 (or 7500 just named as a small_data), that is a set of plays saved as a dict (the key is the sequence_id), and for each play, each row denotes a frame, and columns denote the locations of the players and ball in the frame.

**Comment by IkumaUchida** [comment]
Thank you very much for your reply.

I was able to get a grasp on the contents of `small.pkl`. Perhaps you intentionally created the small size data because the STATS tracking data is large (7500 sequences) and it takes a lot of time when creating a corpus in `building.py`. (I created a corpus with 7500 sequences and spent over a day building the corpus)

This trajectory search method is a very interesting topic, and I would like to borrow some of play2vec's knowledge to devise a new method.

If you don't mind, I would be happy to ask you a few more questions to keep you informed.
